Kernel/Devices/Storage/Management.cpp

```diff
@@ -405,21 +417,22 @@ UNMAP_AFTER_INIT void StorageManagement::determine_boot_device()
 
 UNMAP_AFTER_INIT void StorageManagement::determine_boot_device_with_partition_uuid()
 {
-    VERIFY(!m_storage_devices.is_empty());
```

Comment:
The removal of this check looks incorrect; if we have no devices then we can't determine the boot device.

Reply:
And why is that a problem? if we can't determine a boot device, then it will just fail like with other methods (check `determine_boot_device_with_logical_unit_number`, `determine_ata_boot_device`, etc...).